flutter_appauth/android/src/main/java/io/crossingthestreams/flutterappauth/FlutterAppauthPlugin.java

```diff
@@ -310,6 +310,8 @@ private void performAuthorization(AuthorizationServiceConfiguration serviceConfi
                         clientId,
                         ResponseTypeValues.CODE,
                         Uri.parse(redirectUrl));
+//        authRequestBuilder.setNonce(null);
```

Comment:
As this is commented out, could you remove this instead?

Reply:
Oups, sorry for this left-over 😐